securedrop_salt/sd-proxy.sls

```diff
@@ -25,11 +25,9 @@ sd-proxy-dvm:
       - netvm: sd-whonix
       - template_for_dispvms: True
       - default_dispvm: ""
-  {% if d.environment == "prod" %}
     - features:
       - set:
-        - internal: 1
-  {% endif %}
+        - internal: {{ int(d.environment == "prod") }}
```

Comment:
salt/jinja isn't happy with this syntax so I'm just going to change it to a normal if/else clause.